homeassistant/components/homematicip_cloud/binary_sensor.py

```diff
@@ -104,3 +125,98 @@ def device_class(self):
     def is_on(self):
         """Return true if moisture or waterlevel is detected."""
         return self._device.moistureDetected or self._device.waterlevelDetected
+
+
+class HomematicipSecurityZoneSensorGroup(HomematicipGenericDevice,
+                                         BinarySensorDevice):
+    """Representation of a HomematicIP Cloud water detector."""
+
+    def __init__(self, home, device, post='SecurityZone'):
+        """Initialize heating group."""
+        device.modelType = 'HmIP-{}'.format(post)
+        super().__init__(home, device, post)
+
+    @property
+    def device_class(self):
+        """Return the class of this sensor."""
+        return 'safety'
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the generic device."""
+        attr = super().device_state_attributes
+
+        security_issues = []
+        if self._device.motionDetected is True:
+            security_issues.append(ISSUE_MOTIONDETECTED)
+        if self._device.presenceDetected is True:
+            security_issues.append(ISSUE_PRESENCEDETECTED)
+        from homematicip.base.enums import WindowState
+        if self._device.windowState is not None and \
+                self._device.windowState != WindowState.CLOSED:
+            security_issues.append(ISSUE_WINDOWOPEN)
+
+        if security_issues.__len__() > 0:
+            attr.update({ATTR_SECURITYISSUES: ', '.join(security_issues)})
```

Comment:
This makes it hard to parse the individual issues. I would suggest keeping them separate with one attribute per issue and using `True/False` to mark value.

Reply:
Switched to individual attributes.